### PR TITLE
Improve operation count by add human readable option "-h"

### PR DESCRIPTION
**Pull request by JySongWithZhangCe**
add human readable format option for count
image

### CODE REVIEW

File: shell/src/main/java/alluxio/cli/fs/command/CountCommand.java [hunk 1]
```diff
@@ -33,6 +36,15 @@
 @PublicApi
 public final class CountCommand extends AbstractFileSystemCommand {
 
+  private static final String READABLE_OPTION_NAME = "h";
+
+  private static final Option READABLE_OPTION =
+          Option.builder(READABLE_OPTION_NAME)
+                  .required(false)
+                  .hasArg(false)
+                  .desc("print sizes in human readable format (e.g., 1KB 234MB 2GB)")
```
Comment (jiacheliu3) [inline shell/src/main/java/alluxio/cli/fs/command/CountCommand.java]: `(e.g., 1KB 234MB 2GB)` -> `(e.g. 1KB 234MB 2GB)`?

File: shell/src/main/java/alluxio/cli/fs/command/CountCommand.java [hunk 2]
```diff
@@ -47,7 +59,7 @@ public String getCommandName() {
 
   @Override
   public void validateArgs(CommandLine cl) throws InvalidArgumentException {
-    CommandUtils.checkNumOfArgsEquals(this, cl, 1);
+    CommandUtils.checkNumOfArgsEquals(this, cl, 2);
```
Comment (jiacheliu3) [inline shell/src/main/java/alluxio/cli/fs/command/CountCommand.java]: `-h` is optional so I think the num of args will stay 1. Can you please build Alluxio and give it a manual test?

File: shell/src/main/java/alluxio/cli/fs/command/CountCommand.java (hunk 1 above)
Comment (JySongWithZhangCe) [reply shell/src/main/java/alluxio/cli/fs/command/CountCommand.java]: ok

File: shell/src/main/java/alluxio/cli/fs/command/CountCommand.java (hunk 2 above)
Comment (JySongWithZhangCe) [reply shell/src/main/java/alluxio/cli/fs/command/CountCommand.java]: ok

**Comment by jiacheliu3** [comment]
alluxio-bot, test this please

**Comment by jiacheliu3** [comment]
@LuQQiu Do you mind merging this PR? Thanks!

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Comment by JySongWithZhangCe** [comment]
谢谢嘉承谢谢璐姐！

**Pull request merged**